src/olympia/blocklist/tests/test_models.py

```diff
@@ -64,3 +64,25 @@ def test_is_save_to_blocks_permitted(self):
         # Except when that's not enforced locally
         with override_settings(DEBUG=True):
             assert block.is_save_to_blocks_permitted
+
+    def test_get_submission_from_guid(self):
```

Comment:
For safety's sake, is it worth also having a test that proves that a `guid` that does not exist in a submission returns an empty list?

Reply:
https://github.com/mozilla/addons-server/pull/13479/commits/7595cf2bf80cbc3035ebc93e7ecbe3b1947e81c3